commands/scanpullrequest.go

```diff
@@ -245,17 +246,22 @@ func runInstallAndAudit(scanSetup *utils.ScanDetails, workDirs ...string) (resul
 			return nil, false, err
 		}
 	}
-	auditParams := audit.NewAuditParams().
-		SetXrayGraphScanParams(scanSetup.XrayGraphScanParams).
-		SetServerDetails(scanSetup.ServerDetails).
+	// GraphBasicParams
+	graphBasicParams := &xrayutils.GraphBasicParams{}
+	graphBasicParams.SetPipRequirementsFile(scanSetup.PipRequirementsFile).
 		SetUseWrapper(*scanSetup.UseWrapper).
-		SetRequirementsFile(scanSetup.PipRequirementsFile).
-		SetWorkingDirs(workDirs).
 		SetDepsRepo(scanSetup.Repository).
-		SetReleasesRepo(scanSetup.ReleasesRepo()).
 		SetIgnoreConfigFile(true).
+		SetServerDetails(scanSetup.ServerDetails)
+	// Params
+	auditParams := audit.NewAuditParams()
+	auditParams.SetXrayGraphScanParams(scanSetup.XrayGraphScanParams).
+		SetWorkingDirs(workDirs).
 		SetMinSeverityFilter(scanSetup.MinSeverityFilter()).
-		SetFixableOnly(scanSetup.FixableOnly())
+		SetFixableOnly(scanSetup.FixableOnly()).
+		SetReleasesRepo(scanSetup.ReleasesRepo())
+	auditParams.GraphBasicParams = graphBasicParams
+
```

Comment:
```suggestion
	auditParams := audit.NewAuditParams().
		SetXrayGraphScanParams(scanSetup.XrayGraphScanParams).
		SetWorkingDirs(workDirs).
		SetReleasesRepo(scanSetup.ReleasesRepo()).
		SetMinSeverityFilter(scanSetup.MinSeverityFilter()).
		SetFixableOnly(scanSetup.FixableOnly())
```

Reply:
Here because of the change in the `cli-core`, we have to init GraphBasicParams, or we are crashing due nil pointers.
Or maybe this change should be implemented inside the `cli-core` that will init itself when calling `newAuditParams`.

Reply:
I intended to create the auditParams object in one part. To achieve this, you can merge line 257 with lines 258-264.

Reply:
The thing is that all of the set function from lines 257-261 are returning `*GraphBasicParams`
and the reset are returning `*Params`
that's why i had to split them.
If you have any idea how to combine them that would be great :)